src/app/app-routing.module.ts

```diff
@@ -16,6 +17,10 @@ const routes: Routes = [
     path: Paths.RUNDOWNS,
     loadChildren: () => import('./rundown-overview/rundown-overview.module').then(module => module.RundownOverviewModule),
   },
+  {
+    path: Paths.SETTINGS,
+    loadChildren: () => import('./settings/settings.module').then(m => m.SettingsModule),
```

Comment:
You can change `m` to `module` to make it follow the structure of the other paths.